test/test_wrapper.py

```diff
@@ -670,7 +669,7 @@ def do_something(ctx):
 
     assert counter[0] == 1
 
-
```

Comment:
this test fail on macos without any clear error message

Reply:
Could you make this finer-grained? Decorators can be applied conditionally using function syntax.

Reply:
https://github.com/Czaki/pyopencl/runs/6262916672?check_suite_focus=true#step:4:880 - this is the error that I observed. I do not understand it. But I no longer execute this test during wheel build.